recipes/speclite/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "speclite" %}
+{% set version = "0.7" %}
+{% set sha256 = "2753137e2f3dc0b94d6c3f98c24d674c1a7145410d731cbeb13e0e7f1f7f5ba2" %}
+
+package:
+  version: '{{ version }}'
+  name: '{{ name|lower }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  sha256: {{sha256}}
+  
+build:
+  number: 0
+  noarch: python
```

Comment:
This doesn't look like it can be `noarch: python` since it uses 2to3 and you have a *nix specific `build.sh`:

https://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages

Reply:
You're right, it's true that it needs to be built differently for python 2 and 3. I'll fix this. 

For the build.sh, it is not *nix specific though (it works on mac as well in my tests).